src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final int SALARY_DATE_INDEX = 0;
+    static final int NAME_INDEX = 1;
+    static final int HOURS_WORKED_INDEX = 2;
+    static final int HOURLY_WAGE_INDEX = 3;
+    static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
make `private static final`, we dont want to expose these values
```suggestion
    static final int SALARY_DATE_INDEX = 0;
    static final int NAME_INDEX = 1;
    static final int HOURS_WORKED_INDEX = 2;
    static final int HOURLY_WAGE_INDEX = 3;
    static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```